packages/console/src/hooks/use-confirm-modal.ts

```diff
@@ -0,0 +1,23 @@
+import { useCallback } from 'react';
+import { create } from 'react-modal-promise';
+
+import { ConfirmModalProps } from '@/components/ConfirmModal/ConfirmModal';
+import ConfirmPromiseModal from '@/components/ConfirmModal/ConfirmPromiseModal';
+
+type UseConfirmModalProps = Omit<ConfirmModalProps, 'isOpen' | 'onCancel' | 'onConfirm'>;
+
+const useConfirmModal = () => {
+  const confirm = useCallback(async (props: UseConfirmModalProps) => {
+    try {
+      await create(ConfirmPromiseModal)(props);
+
+      return true;
+    } catch {
+      return false;
+    }
```

Comment:
What's the return value for? Do we have to return a boolean value?

Reply:
This is used to know whether the user is confirmed.

Reply:
Oh, I see. You need this boolean value to determine if the user has confirmed or not. Hmmm